src/main/java/it/infuse/jenkins/usemango/UseMangoBuilder.java

```diff
@@ -290,19 +314,38 @@ public FormValidation doCheckProjectId(@QueryParameter String value)
 			}
         }
 
+		@POST
+		public FormValidation doCheckEnvironmentId(@QueryParameter String projectId)
+				throws IOException, UseMangoException {
+			try {
+				if(!projectId.isEmpty()) {
+					String defaultEnvironmentName = getDefaultEnvironment(projectId).getName();
+					return FormValidation.okWithMarkup("Default Environment: " + defaultEnvironmentName);
+				}
+				return FormValidation.ok();
+			}
+			catch (HttpResponseException e) {
+				return FormValidation.warning("Some error occurred");
```

Comment:
Do we want to clarify what the error is, some more info would be helpful to the user